src/reducers/userMessages.js

```diff
@@ -8,6 +8,7 @@ import {
   SET_USER_MESSAGES,
 } from '_/constants'
 import { actionReducer } from './utils'
+import uniqueId from 'lodash/uniqueId'
```

Comment:
Why not just use:
```js
import { generateUnique } from '_/helpers'
```

Reply:
Because `Math.random` could return same numbers, but `uniqueId` use `idCounter` variable so, id will be unique within one session.

Reply:
🤷‍♂ 

`generateUnique` actually follows the UUID v4 algo.  Shouldn't be any problem there.

Reply:
Yes, chance that 2 UUID will be same is very small, but I prefer to leave `uniqeId`, because in this case chance is 0, and we already using lodash.

Reply:
Makes sense.